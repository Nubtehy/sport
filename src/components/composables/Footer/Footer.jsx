
import React from 'react';
import FooterWrapper from './Footer.styled';
import { CartContainer } from '../Master/Master.styled';

const displayName = 'Footer';

const propTypes = {
};

const defaultProps = {

};

function Footer() {
  return (
    <FooterWrapper>
      <CartContainer>
          @onSEO
      </CartContainer>
    </FooterWrapper>
  );
}


Footer.displayName = displayName;
Footer.propTypes = propTypes;
Footer.defaultProps = defaultProps;

export default Footer;
