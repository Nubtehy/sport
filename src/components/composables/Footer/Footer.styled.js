import styled from 'styled-components';

const FooterWrapper = styled.footer`
  flex: 0 0 auto;
  background: #cccccc;
  padding: 10px;
`;
export default FooterWrapper;
