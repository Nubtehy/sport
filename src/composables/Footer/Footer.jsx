
import React from 'react';


const displayName = 'Footer';

const propTypes = {
};

const defaultProps = {

};

function Footer() {
  return (
    <header>
      <div>
        This is Footer
      </div>
    </header>
  );
}


Footer.displayName = displayName;
Footer.propTypes = propTypes;
Footer.defaultProps = defaultProps;

export default Footer;
