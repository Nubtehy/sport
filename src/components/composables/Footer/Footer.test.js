import React from 'react';
import { shallow } from 'enzyme';

import Footer from './Footer';

const testProps = {
  dataQa: 'parent-form-dataQa',
};

describe('Given a Footer component', () => {
  describe('when the component is rendered', () => {
    let component;

    beforeEach(() => {
      component = shallow(<Footer {...testProps} />);
    });

    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
