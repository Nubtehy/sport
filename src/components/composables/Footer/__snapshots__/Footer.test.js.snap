// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Footer component when the component is rendered should match the snapshot 1`] = `
<Footerstyled__FooterWrapper>
  <Masterstyled__CartContainer>
    @onSEO
  </Masterstyled__CartContainer>
</Footerstyled__FooterWrapper>
`;
